dts/common/nordic/nrf54l09.dtsi

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "nrf54l_05_10_15.dtsi"
+
+/delete-node/ &gpio2;
+/delete-node/ &spi00;
+/delete-node/ &uart00;
+/delete-node/ &spi22;
+/delete-node/ &i2c22;
+/delete-node/ &uart22;
+/delete-node/ &pdm20;
+/delete-node/ &pdm21;
+/delete-node/ &pwm20;
+/delete-node/ &pwm21;
+/delete-node/ &pwm22;
+/delete-node/ &i2s20;
+/delete-node/ &qdec20;
+/delete-node/ &qdec21;
+/delete-node/ &nfct;
```

Comment:
The `nrf54l_05_10_15.dtsi` was added due to the fact that those are identical HW wise, except for memory.
L09 is not, so I think it should not depend on that file.

Reply:
Removed the dependency and defined all nodes from scratch in `nrf54l09.dtsi`